src/content/reference/cli.mdx

```diff
@@ -383,7 +383,7 @@ Use the arrow keys to navigate: ↓ ↑ → ←
 
 #### create
 
-Create a namespace.
+Create a namespace. By default the command will switch to the new namespace, add `--use=false` for just creating the namespace and keeping the current one active.
```

Comment:
```suggestion
Create a namespace. By default the command will switch to the new namespace, add `--use=false` to create the namespace, and it will keep the current namespace active.
```

Non-blocking suggestion based on what I think I understand of this copy 😄